srcpkgs/cheat/INSTALL.msg

```diff
@@ -0,0 +1,10 @@
+cheatsheets aren't provided and should be set at a suitable location:
```

Comment:
INSTALL.msg is meant for hw-bricking class of issues, not hand-holding. Install with vdoc or remove.